### ISSUE TITLE
"Cannot find name" error if helper and block param names start with same character(s)

### ISSUE THREAD

**Issue by nwhittaker** [typed-ember/glint#255]
Running `npx glint` on the following template that includes both block params and the standard `ember-intl` [translation helper][1]:

```hbs
<MyComponent as |title|>
  {{title}}
  {{t 'translation.key-1'}}
</MyComponent>
{{t 'translation.key-2'}}
```

Results in the message:
```
error TS2304: Cannot find name 't'.

  {{t 'translation.key-1'}}
```

Interestingly, if I change the name of the block param to something that does not start with `t`, no error occurs. In either case, the strings are translated correctly, so it appears as though the linter is flagging the issue incorrectly.

[1]: https://github.com/Gavant/glint-template-types/blob/main/types/ember-intl/translation-helper.d.ts

**Comment by boris-petrov** [comment]
This is a duplicate of #182 which was recently fixed and will be in the next release.

**Comment by dfreeman** [comment]
This is fixed in 0.7.0, which went out earlier today!
